bftengine/src/bftengine/SigManager.hpp

```diff
@@ -66,7 +66,7 @@ class SigManager {
     metrics_component_.SetAggregator(aggregator);
   }
   void setClientPublicKey(const std::string& key, PrincipalId, KeyFormat);
-
+  bool hasVerifier(PrincipalId pid);
```

Comment:
Is it a valid situation when there's no verifier for a client in `SigManager`?

Reply:
Yes, the operator for example. 
I've added this only for having more cleat logs (every verification failure prints a log line)